File: Coding/Ruby/mult_tables.rb
# Program to get a number n from the user and then output the multiplication tables from 1 to 12 for that number

puts "Hello user please enter the number n: "
n = gets.to_i

for i in 1 .. 12
    puts "#{n}\tx\t#{i}\t=\t#{n*i}"
end

File: Coding/C++/mult_tables.cpp
/* Program that asks the user for a number and calculates the multiplication tables for the entered number */

#include<iostream>
#include<math.h>
using namespace std;

int main(){
    int n,i;

    cout<<"Hello user, please enter the number n: ";
    cin>>n;

    for (i = 1; i <= 12; i++){
        cout<< n << "\tx\t" << i << "\t=\t" << n*i << "\n";
    }

    return 0;
}